components/ProjectPicker.tsx

```diff
@@ -14,16 +14,18 @@ const ProjectPicker = ({ id }): JSX.Element => {
   const router = useRouter();
   const dispatch = useDispatch();
 
-
   useEffect(() => {
     if (!projects.length) {
       //todo: move this out to an action
       const getProjects = async () => {
-        const result = await fetch('https://www.pivotaltracker.com/services/v5/projects', {
-          headers: {
-            'X-TrackerToken': apiToken,
-          },
-        });
+        const result = await fetch(
+          'https://www.pivotaltracker.com/services/v5/projects',
```

Comment:
```suggestion
          'https://www.pivotaltracker.com/services/v5/projects',
```
Should this be an env variable in case we see shared usage later on?

Reply:
👍  can do as follow-up PR